pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.alfresco</groupId>
     <artifactId>api-explorer</artifactId>
-    <version>5.2.f-SNAPSHOT</version>
+    <version>5.2.1-SNAPSHOT</version>
```

Comment:
Remember to update the master branch back 5.2.f-SNAPSHOT after this gets merged! You can do that change directly on master.